jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -121,11 +122,14 @@ public void init() {
     @OnOpen
     public void onOpen() {
         super.onOpen();
-        setFilterSettings(filterSettingsManager.createDefaultFilterSettingsPrototype(),
-                          table -> {
-                              setupActiveSearchFilters();
-                              addDataDisplay(table);
-                          });
+        filterSettingsManager.defaultActiveFilterInit(filterSettings -> addActiveFilters(filterSettings),
```

Comment:
filter manager should handle the default filter.... if one doesnt exist it should consider creating it, based on the default filter we already apply.